cpp/include/cudf/io/detail/utils.hpp

```diff
@@ -25,5 +26,16 @@ namespace io::detail {
  */
 enum class single_write_mode : bool { YES, NO };
 
+template <typename T>
+constexpr static bool is_not_handled()
```

Comment:
why not 
```suggestion
constexpr static bool is_handled()
```
so we can avoid double negatives in the code?

Reply:
Done